pkg/clients/packet.go

```diff
@@ -25,7 +25,7 @@ import (
 
 // NewClient ... TODO
 func NewClient(ctx context.Context, credentials []byte) *packngo.Client {
-	return packngo.NewClient("crossplane", string(credentials), nil)
+	return packngo.NewClientWithAuth("crossplane", string(credentials), nil)
```

Comment:
Was this a change in `packngo` v0.2.0?

Reply:
Not just that one. UpdateDeviceRequest was not mapped correctly before. 
```
// DeviceUpdateRequest type used to update a Packet device
type DeviceUpdateRequest struct {
	Hostname      string   `json:"hostname,omitempty"`
```
vs
```
// DeviceUpdateRequest type used to update a Packet device
type DeviceUpdateRequest struct {
	Hostname      *string   `json:"hostname,omitempty"`
```